module/x/gravity/abci.go

```diff
@@ -35,7 +35,14 @@ func createValsets(ctx sdk.Context, k keeper.Keeper) {
 
 	significantPowerDiff := false
 	if latestValset != nil {
-		intCurrMembers, err := types.BridgeValidators(k.GetCurrentValset(ctx).Members).ToInternal()
+		vs, err := k.GetCurrentValset(ctx)
```

Comment:
As discussed in my other comment, if we're making GetCurrentValset() return an error it should not panic and return two different types of errors, from there we can identify the empty valset one explicitly and log it here. I would also like to further guard for this by checking that we are in a simulation. Is there any way to do that?

Reply:
I don't believe so, although I'll look into it